ui/rewards/MeritsIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon, useColorModeValue, chakra } from '@chakra-ui/react';
+import React from 'react';
+
+// This icon doesn't work properly when it is in the sprite
+// Probably because of the gradient
+// eslint-disable-next-line no-restricted-imports
+import meritsIcon from 'icons/merits_colored.svg';
```

Comment:
Remind me why we didn't put it in the static files and use as image? 🤔

Reply:
Same question for other icons with the same comment 🤔